src/osd/OSDMap.cc

```diff
@@ -7330,6 +7330,15 @@ void OSDMap::check_health(CephContext *cct,
       ss << down_in_osds.size() << " osds down";
       auto& d = checks->add("OSD_DOWN", HEALTH_WARN, ss.str(),
 			    down_in_osds.size());
+      bool crimson_build_but_flags_disabled = cmd_getval_or<bool>(cmdmap, "crimson", false) &&
+               !cct->_conf->get_val<bool>("osd_pool_default_crimson");
```

Comment:
My thinking here was that I want to raise the warning if we're running a crimson build, but the flag's not set. Maybe this "cmdmap" crimson value is not the one I want? I saw other options like `experimental_features_enabled`, and/or 'ifdef SEASTAR', but they seemed far fetched...